cmd/service_bootstrap_desktop.go

```diff
@@ -164,16 +166,40 @@ func (di *Dependencies) bootstrapServiceComponents(nodeOptions node.Options) {
 	di.ServiceRegistry = service.NewRegistry()
 	di.ServiceSessionStorage = session.NewStorageMemory()
 
-	newDialogWaiter := func(providerID identity.Identity, serviceType string) (communication.DialogWaiter, error) {
+	registeredIdentityValidator := func(peerID identity.Identity) error {
+		registered, err := di.IdentityRegistry.IsRegistered(peerID)
+		if err != nil {
+			return err
+		} else if !registered {
```

Comment:
the else is redundant

Reply:
It's not. 
It's `else if ` just for a smaller number of lines instead of two `if`. :)